easyInterface/Diffraction/DataClasses/Utils/BaseClasses.py

```diff
@@ -92,39 +94,57 @@ def __init__(self, value: Optional[Any] = None, unit: Optional[Union[str, Unit]]
         if not isinstance(unit, Unit):
             # Try to convert the unit to a string
             unit = Unit(unit)
-        super().__init__(value=value, unit=unit, error=0, constraint=None, hide=True, refine=False)
-        self._log = logging.getLogger(__name__)
-        self._log.log(VERBOSE, 'Data object created with default value %s, unit %s', value, unit)
+        super().__init__(value=value, unit=unit, min=-np.Inf, max=np.Inf, error=0, constraint=None, hide=True, refine=False)
+        self._log = logger.getLogger(__class__.__module__)
 
     def __repr__(self) -> str:
         return '{}'.format(self['value'])
 
     @property
     def min(self) -> float:
-        value = self['value']
-        ret = None
-        if value is not None:
-            if isinstance(value, (int, float, complex)) and not isinstance(value, bool):
-                ret = 0.8 * self['value']
-        return ret
+        return self['min']
+
+    @min.setter
+    def min(self, value):
+        self['min'] = value
 
     @property
     def max(self) -> float:
-        value = self['value']
-        ret = None
-        if value is not None:
-            if isinstance(value, (int, float, complex)) and not isinstance(value, bool):
-                ret = 1.2 * self['value']
-        return ret
+        return self['max']
 
+    @max.setter
+    def max(self, value):
+        self['max'] = value
 
 class Base(LoggedPathDict):
     def __init__(self, value: object = None, unit: object = '') -> object:
         super().__init__(header='Undefined', tooltip='', url='', mapping=None, store=Data(value, unit))
+        self._log = logger.getLogger(__class__.__module__)
+        self.updateMinMax()
 
     def __repr__(self) -> str:
         return '{} {}'.format(self.value, self.getItemByPath(['store', 'unit']))
 
+    def updateMinMax(self):
+        self._log.debug(f"value: {self.value}")
```

Comment:
debug or VERBOSE level? And, a more informative text

Reply:
Debug logging is removed